sysinfo.ts

```diff
@@ -64,6 +66,26 @@ export class SysInfo implements ISysInfo {
 		} // if we got an error, assume not working
 	}
 
+	// `android -h` returns exit code 1 on successful invocation. Therefore, we cannot use $childProcess
+	private execAndroidH(): string {
```

Comment:
can you change the return type to IFuture<string> and return the future you create on the next line? I suggest this only because we stick to the idea each `.wait` to be wrapped in IFuture :)